flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/RestoreTestBase.java

```diff
@@ -248,7 +247,7 @@ private void registerSinkObserver(
      * Execute this test to generate test files. Remember to be using the correct branch when
      * generating the test files.
      */
-    @Disabled
+    // @Disabled
```

Comment:
is this an intentional change?
Enabling a test is obviously a good idea but if that's intended, we should IMO remove the attachment completely not just comment it out.